lexer_schemas/commerce_api/product_entity.py

```diff
@@ -74,6 +76,14 @@ class ProductRecord(BaseModel):
         examples=[["https://fake.com/images/menswear/sawyer-rib-crew-knit.jpg"]],
         default=None,
     )
+    availability: Optional["ProductAvailability"] = Field(
+        title="Product Availability",
+        default=None,
+    )
+    inventory: Optional["ProductInventory"] = Field(
+        title="Product Inventory",
+        default=None,
+    )
```

Comment:
Need some descriptions here

Reply:
This RFC could help you with field descriptions / creating plausible examples
https://www.notion.so/lexer/RFC-Inventory-Data-in-CDE-05025371443a425b9ac8e4739e8357d1